sti/karaf/Dockerfile

```diff
@@ -38,10 +41,11 @@ COPY ./jolokia.properties /opt/jolokia/jolokia.properties
 COPY ./deploy-and-run.sh /opt/karaf/bin/
 RUN chmod a+x /opt/karaf/bin/deploy-and-run.sh
 
-# allow jboss user access to /opt/jboss
-RUN chown -R 1000 /opt/jboss
+# allow jboss user access to /opt/jboss and /opt/karaf
+RUN chown -R 1000 /opt/jboss \
```

Comment:
Yeah, that's being replaced with making the dirs world writable according to openshift guidelines